satpy/readers/gerb_l2_hr_h5.py

```diff
@@ -48,7 +61,26 @@ def gerb_get_dataset(ds, ds_info):
 
 
 class GERB_HR_FileHandler(HDF5FileHandler):
-    """File handler for GERB L2 High Resolution H5 files."""
+    """File handler for GERB L2 High Resolution H5 files.
+
+    **Overriding the ``area_def``**
+
+    By default, the GERB HR reader looks at the attribute "Geolocation/attr/Nominal
+    Satellite Longitude (degrees)" for selecting the area definition. This data is not
+    available for the GERB-like products however, so you may need to override it with the
+    ``area_def`` argument::
+
+        scene = satpy.Scene(filenames, reader="gerb_l2_hr_h5",
+                                       area_def="msg_seviri_fes_9km")
+
+    """
+
+    def __init__(self, filename, filename_info, filetype_info, area_def=None):
```

Comment:
Thoughts on `area` instead of `area_def`?

Reply:
Makes sense.